### ISSUE TITLE
Failed to execute 'setAttribute' on 'Element': '{name}' is not a valid attribute name.

### ISSUE THREAD

**Issue by plrenaudin** [elcobvg/svelte-autocomplete#4]
Hi, I tried to use your component just with the minimum requirement:
```
<AutoComplete class="input" name="fruits" items="{fruits}" minChar="1" />
```

But it fails at runtime:

```
h:function hydrate() {
			addListener(input, "input", input_input_handler);
			addListener(input, "input", input_handler);
			addListener(input, "focus", focus_handler);
			addListener(input, "blur", blur_handler);
			addListener(input, "keydown", keydown_handler);
			setAttribute(input, "type", "text");
			input.className = "{class} svelte-1yo2r3";
			setAttribute(input, "{name}", true);
			setAttribute(input, "{placeholder}", true);
			setAttribute(input, "{required}", true);
			setAttribute(input, "{disabled}", true);
			input.value = "{value || ''}";
			input.autocomplete = "{name}";
			addListener(li, "click", click_handler);
			li.className = "autocomplete-result{ i === arrowCounter ? ' is-active' : '' } svelte-1yo2r3";
			ul.className = "autocomplete-results{!isOpen ? ' hide-results' : ''} svelte-1yo2r3";
			if (!slot_content_default) {
				p.className = "fallback svelte-1yo2r3";
			}
			addListener(div, "click", click_handler_1);
			div.className = "autocomplete svelte-1yo2r3";
		},
```
 
Happens on the `setAttribute(input, "{name}", true);` line

I am using svelte 2.11 and parcel 1.9.7 with the svelte plugin 1.2.2

**Comment by elcobvg** [comment]
OK, will have a look at this as soon as I have the time.

**Comment by antony** [comment]
I get the same error, sadly.

**Comment by antony** [comment]
@plrenaudin Actually, I think we're doing something wrong. Probably you don't have `items` in your component data.

The component works fine:

https://svelte.technology/repl?version=2.13.4&gist=a0896d1dde2ca791e2988ff64d29f344

**Comment by antony** [comment]
After a little further diagnosis, I believe this might be a parceljs issue. I too am using Parcel.

My example from the REPL doesn't work if I add it to my project.

**Comment by elcobvg** [comment]
It may well be a Parcel issue. I must admit I've never tested it with Parcel.

**Comment by elcobvg** [comment]
Closed #7
